opnsynid_receivable_aging_aeroo_report/report/query_receivable_aging.py

```diff
@@ -0,0 +1,228 @@
+# -*- coding: utf-8 -*-
+# © 2015 OpenSynergy Indonesia
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, fields, api
+from openerp.tools import drop_view_if_exists
+from datetime import datetime
+
+
+class QueryReceivableAging(models.Model):
+
+    _name = 'account.query_receivable_aging'
+    _description = 'Query Receivable Aging'
+    _auto = False
+
+    @api.one
+    def function_aging(self):
+        res = {}
+        residual = 0.0
+        obj_move_line = self.env['account.move.line']
+        period_length = self._context.get('period_length', False)
```

Comment:
Oke dicoba aja dulu. Gw malah baru tau pake ._context.

On Fri, Apr 22, 2016 at 3:22 PM, mikevhe18 <notifications@github.com> wrote:

> In opnsynid_receivable_aging_aeroo_report/report/query_receivable_aging.py
> <https://github.com/open-synergy/opnsynid-accounting-report/pull/55#discussion_r60705523>
> :
>
> > +from openerp.tools import drop_view_if_exists
> > +from datetime import datetime
> > +
> > +
> > +class QueryReceivableAging(models.Model):
> > +
> > +    _name = 'account.query_receivable_aging'
> > +    _description = 'Query Receivable Aging'
> > +    _auto = False
> > +
> > +    @api.one
> > +    def function_aging(self):
> > +        res = {}
> > +        residual = 0.0
> > +        obj_move_line = self.env['account.move.line']
> > +        period_length = self._context.get('period_length', False)
>
> itu dari referensi yg mas kasi, sama dari contoh2 modul yg pake @api
> (account.invoice)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/open-synergy/opnsynid-accounting-report/pull/55/files/f0d5ab0d93345a64fca652cb8f27f27a42bd51a2#r60705523>
>



-- 


[image: --]

Andhitia Rama
[image: https://]about.me/andhitia.r
<https://about.me/andhitia.r?promo=email_sig>